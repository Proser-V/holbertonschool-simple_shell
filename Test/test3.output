README.md
Shared_diagram-VD.jpg
Test
VD_tests
XL_tests
builtin.c
exec.c
exit.c
generate-authors.sh
helper.c
hsh
main.c
more_alloc.c
path.c
read.c
shell.h
split.c
test
test.sh
test_shell.c
free(): double free detected in tcache 2
