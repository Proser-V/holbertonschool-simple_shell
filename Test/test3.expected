README.md
Test
XL_tests 
exec.c
generate-authors.sh
hsh
more_alloc.c
read.c
split.c  
test.sh
Shared_diagram-VD.jpg
VD_tests
builtin.c
exit.c
helper.c
main.c
path.c
shell.h
test
test_shell.c